src/main/java/seedu/duke/lesson/Lesson.java

```diff
@@ -38,4 +59,43 @@ public void setTeachingStaff(TeachingStaff teachingStaff) {
     public void setOnlineLink(String onlineLink) {
         this.onlineLink = onlineLink;
     }
+
+    public LessonType initialiseLessonType(String lessonType) {
+        switch (lessonType) {
+        case TUTORIAL:
+            return LessonType.TUTORIAL;
+
+        case LECTURE:
+            return LessonType.LECTURE;
+
+        case LAB:
+            return LessonType.LAB;
+
+        default:
+            //throw required exception if parser didn't handle
+            System.out.println("Wrong name");
+            return LessonType.LECTURE; //temporary
+        }
+    }
+
+    /**
+     * User input: "add tutorial ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc"
+     * Parser input: "tutorial ;; Wednesday 9 am - 10am ;; https://nus-sg.zoom.us/j/abc ;;  ;; "
+     * Parser will input " " and delimiters in parser input equal to five i,e
+     * Tentatively discussed with Ivan(REMOVE THIS COMMENT AFTER IMPLEMENTATION)
+     * This method add a new lesson to the list of lessons of a module
```

Comment:
Might need to modify this to meet the Java Coding Standards.

Reply:
This is not the final comment. I wrote this for Ivan. Will change it after he's done implementing the parser. No worries